src/tests/decider-distro-tests-rs/tests/utils/default.rs

```diff
@@ -24,8 +24,8 @@ pub fn default_receipt() -> Value {
 
 pub const DEAL_STATUS_ACTIVE: &str =
     "0x0000000000000000000000000000000000000000000000000000000000000001";
-pub const DEAL_STATUS_INACTIVE: &str =
-    "0x0000000000000000000000000000000000000000000000000000000000000000";
+pub const DEAL_STATUS_NOT_ENOUGH_WORKERS: &str =
+    "0x0000000000000000000000000000000000000000000000000000000000000003";
```

Comment:
it won't work, decider expects 0 for inactive deal

Reply:
Oh, nevermind

Reply:
I fixed it

Reply:
aaah in tests it won't work, will fix it, thx

Reply:
no, it will, you've changed decider, so inactive state is 3

Reply:
i need to fix it in mocks